products/kubernetes.md

```diff
@@ -3,96 +3,125 @@ title: Kubernetes
 category: server-app
 iconSlug: kubernetes
 permalink: /kubernetes
+alternate_urls:
+-   /k8s
 versionCommand: kubectl version
 releasePolicyLink: https://kubernetes.io/releases/patch-releases/
 releaseImage: https://upload.wikimedia.org/wikipedia/en/timeline/hwofyi7fnfgyjzfs6er9ha6mvxnakw7.png
 changelogTemplate: https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-__RELEASE_CYCLE__.md
 releaseDateColumn: true
 activeSupportColumn: true
 eolColumn: Maintenance Support
+
 auto:
 -   git: https://github.com/kubernetes/kubernetes.git
     regex: ^v(?<major>[1-9]\d*)\.(?<minor>0|[1-9]\d*)\.(?<patch>0|[1-9]\d*)$
-alternate_urls:
--   /k8s
-# The release date for "N" should match the eol date for N-3 release.
+
+# support(x) = releaseDate(x) + 12 months
+# support(x) = releaseDate(x) + 14 months
 releases:
+-   releaseCycle: "1.27"
+    releaseDate: 2023-04-11
+    support: 2024-04-11
+    eol: 2024-06-11
```

Comment:
According to https://kubernetes.io/releases/patch-releases/#1-27
> 1.27 enters maintenance mode on 2024-04-28 and End of Life is on 2024-06-28.

```suggestion
    support: 2024-04-28
    eol: 2024-06-28
```

Reply:
Focused on tables and missed the end of support / eol dates on that page. Good catch @BiNZGi !

Will update the comment too.